src/scenes/home/mentorRequest/mentorRequest.css

```diff
@@ -0,0 +1,80 @@
+.mentorRequest {
+  width: 100%;
+}
+
+.formBlock {
+  width: 50%;
+  padding: 0 1rem;
+}
+
+.formElement {
+  height: 11rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.formRow {
+  width: 100%;
+  padding: 1rem;
+}
+
+.formRow input {
+  width: 100%;
+}
+
+.formElement p {
+  margin: 0;
+}
+
+.mentorRequestForm span {
+  padding: 10px 8px 30px 8px;
+  font-size:16px;
+}
+
+.mentorRequestForm {
+  max-width: 90%;
+  margin: auto;
+  border: 1px solid rgb(92,105,122);
+  padding: 10px;
+  height: 100%;
+  display: block;
+  width: 60rem;
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+}
+
+.mentorRequestForm select {
+  height: 4.2rem;
+  width: 100%;
+  padding: 2rem;
```

Comment:
The vertical padding is preventing the selected option from being visible on Chrome for Windows.  Recommend changing the padding to `0 16px` (or the rem equivalent) to mirror the styling on the form inputs.